salesforce-connector/src/com/axonivy/connector/salesforce/utils/ConvertUtils.java

```diff
@@ -0,0 +1,86 @@
+package com.axonivy.connector.salesforce.utils;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.beanutils.BeanUtils;
+
+import com.axonivy.connector.salesforce.model.Account;
+import com.axonivy.connector.salesforce.model.Event;
+import com.axonivy.connector.salesforce.model.Opportunity;
+import com.axonivy.connector.salesforce.model.OpportunityUpdateDTO;
+import com.axonivy.connector.salesforce.model.Task;
+import com.axonivy.connector.salesforce.response.CreateOppResponse;
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.util.BeanUtil;
+
+import ch.ivyteam.ivy.process.call.SubProcessCall;
+
+public class ConvertUtils {
+	
+	public static Opportunity convertToOpportunity(JsonNode jsonNode) throws StreamReadException, DatabindException, IOException {
+		ObjectMapper mapper = new ObjectMapper();
```

Comment:
ObjectMapper will take some more resources on the first time it is initialized, so it's recommended to create one object and reuse it. @HoDacVu90 
https://stackoverflow.com/questions/57670466/objectmapper-best-practice-for-thread-safety-and-performance